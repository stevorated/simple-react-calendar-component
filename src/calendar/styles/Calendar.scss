.calendarContainer {
  padding: 0;
  margin: 0;
}

.calendarTitle {
  font-size: 40px;
}

.monthTitle {
  font-size: 30px;
  margin-top: 10px;
  margin-bottom: 10px;
}