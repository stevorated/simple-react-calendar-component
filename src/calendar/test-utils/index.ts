export const findByTestAttr = (component, attr) => {
  return component.find(`[data-test="${attr}"]`)
}


export const findByTestAttrElement = (component, attr, element = 'div') => {
  return component.find(`${element}[data-test="${attr}"]`)
}