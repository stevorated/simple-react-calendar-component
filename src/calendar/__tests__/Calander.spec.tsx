import React from 'react';
import Calendar, { DataObj } from '../../index';
import renderer from 'react-test-renderer';
import { findByTestAttr, data, findByTestCustom } from '../test-utils';
import { mount } from 'enzyme';

let component;
const title = 'ya alla';

const setupRenderer = (props = {}) => {
  return renderer.create(
    <Calendar
      targetDay={1}
      targetDateString="2019-10-01"
      targetMonth="2019-10-01"
      {...props}
    />
  );
};

const setupMountRender = (props = {}) => {
  return mount(
    <Calendar
      targetDay={1}
      targetDateString="2019-10-01"
      targetMonth="2019-10-01"
      {...props}
    />
  );
};

describe('<Calendar />', () => {
  test('should match snapshot', () => {
    let rendered = setupRenderer({});
    const tree = rendered.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should match snapshot with data and titles', () => {
    let rendered = setupRenderer({ data, title, showMonth: true });
    const tree = rendered.toJSON();
    expect(tree).toMatchSnapshot();
  });

  test('should shallow render correctly with minimal props', () => {
    component = setupMountRender({});
    const calendarContainer = findByTestAttr(component, 'calendarContainer');
    const calendarTitle = findByTestAttr(component, 'calendarTitle');
    const monthTitle = findByTestAttr(component, 'monthTitle');
    const calendarHeaders = findByTestAttr(component, 'calendarHeaders');

    expect(calendarContainer.length).toBe(1);
    expect(calendarTitle.length).toBe(1);
    expect(calendarTitle.text()).toBe('');
    expect(monthTitle.length).toBe(0);
    expect(calendarHeaders.length).toBe(1);
  });

  test('should shallow render correctly with minimal props and titles', () => {
    component = setupMountRender({
      title,
      showMonth: true,
    });
    const calendarTitle = findByTestAttr(component, 'calendarTitle');
    const monthTitle = findByTestAttr(component, 'monthTitle');

    expect(calendarTitle.length).toBe(1);
    expect(calendarTitle.text()).toBe(title);
    expect(calendarTitle.props().className).toBe('calendarTitle' + ' ');

    expect(monthTitle.length).toBe(1);
    expect(monthTitle.props().className).toBe('monthTitle' + ' ');
  });

  test('should shallow render custom title classes', () => {
    const titleContainerClass = 'exampleTitleContainerClass';
    const monthTitleClass = 'exampleMonthTitleClass';
    const containerClassName = 'exampleClassContainer';
    const rowContainerClassName = 'exampleClassRow';
    const dayContainerClassName = 'exampleClassDay';
    const dayDataListClass = 'exampleDayDataListClass';
    const dayDataListItemClass = 'exampleDayDataListItemClass';
    const daysHeaderContainerClass = 'exampleDaysHeaderContainerClass';
    const daysTitleContainerClass = 'exampleDaysTitleContainerClass';

    component = setupMountRender({
      title,
      data,
      showMonth: true,
      titleContainerClass,
      monthTitleClass,
      containerClassName,
      rowContainerClassName,
      dayContainerClassName,
      dayDataListClass,
      dayDataListItemClass,
      daysHeaderContainerClass,
      daysTitleContainerClass,
    });
    const calendarContainer = findByTestAttr(component, 'calendarContainer');
    const calendarRowContainer = findByTestAttr(
      component,
      'calendarRowContainer'
    );
    const calendarDayContainer = findByTestAttr(
      component,
      'calendarDayContainer'
    );
    const calendarTitle = findByTestAttr(component, 'calendarTitle');
    const monthTitle = findByTestAttr(component, 'monthTitle');
    const dayDataList = findByTestAttr(component, 'dayDataList');
    const dayDataListItem = findByTestAttr(component, 'dayDataListItem');
    const calendarDaysContainer = findByTestAttr(
      component,
      'calendarDaysContainer'
    );
    const calendarHeaderContainer = findByTestAttr(
      component,
      'calendarHeaderContainer'
    );

    expect(calendarDaysContainer.first().props().className).toBe(
      'calendarDaysContainer' + ' ' + daysHeaderContainerClass
    );

    expect(calendarHeaderContainer.first().props().className).toBe(
      'calendarHeaderContainer' + ' ' + daysTitleContainerClass
    );

    expect(calendarContainer.props().className).toBe(
      'calendarContainer' + ' ' + containerClassName
    );
    expect(calendarRowContainer.first().props().className).toBe(
      'calendarRow' + ' ' + rowContainerClassName
    );
    expect(calendarDayContainer.first().props().className).toBe(
      'calendarDayContainer' + ' ' + ' ' + dayContainerClassName
    );
    expect(calendarTitle.length).toBe(1);
    expect(calendarTitle.text()).toBe(title);
    expect(calendarTitle.props().className).toBe(
      'calendarTitle' + ' ' + titleContainerClass
    );

    expect(dayDataList.first().props().className).toBe(
      'dayDataList' + ' ' + dayDataListClass
    );

    expect(dayDataListItem.first().props().className).toBe(
      'dayDataItem' + ' ' + dayDataListItemClass
    );

    expect(monthTitle.length).toBe(1);
    expect(monthTitle.props().className).toBe(
      'monthTitle' + ' ' + monthTitleClass
    );
  });

  test('should shallow render October correct row, days and days with dates', () => {
    component = setupMountRender({});
    const calendarRow = findByTestAttr(component, 'calendarRow');
    const calendarRowContainer = findByTestAttr(
      component,
      'calendarRowContainer'
    );
    const calendarDay = findByTestAttr(component, 'calendarDay');
    const calendarNum = findByTestAttr(component, 'calendarNum');

    expect(calendarRow.length).toBe(5);
    expect(calendarRowContainer.length).toBe(5);
    expect(calendarDay.length).toBe(35);
    expect(calendarNum.length).toBe(31);
  });

  test('should shallow render February correct row, days and days with dates', () => {
    component = setupMountRender({
      targetDay: 1,
      targetDateString: '2019-02-01',
      targetMonth: '2019-02-01',
    });
    const calendarRow = findByTestAttr(component, 'calendarRow');
    const calendarRowContainer = findByTestAttr(
      component,
      'calendarRowContainer'
    );
    const calendarDay = findByTestAttr(component, 'calendarDay');
    const calendarNum = findByTestAttr(component, 'calendarNum');
    const dayDataList = findByTestAttr(component, 'dayDataList');

    expect(calendarRow.length).toBe(5);
    expect(calendarRowContainer.length).toBe(5);
    expect(calendarDay.length).toBe(35);
    expect(calendarNum.length).toBe(28);
    expect(dayDataList.length).toBe(0);
  });

  test('should render data prop correctly', () => {
    component = setupMountRender({
      data,
    });
    const dayDataList = findByTestAttr(component, 'dayDataList');
    const dayDataListItem = findByTestAttr(component, 'dayDataListItem');

    expect(dayDataList.length).toBe(35);
    expect(dayDataListItem.length).toBe(data.length);
  });

  test('should render active day class with color', () => {
    component = setupMountRender({ colorActiveDate: 'blue' });
    const activeDay = findByTestCustom(
      component,
      'data-test2',
      'calanderActiveDate'
    );
    expect(activeDay.props().style).toEqual({
      backgroundColor: 'blue',
    });
  });
});
