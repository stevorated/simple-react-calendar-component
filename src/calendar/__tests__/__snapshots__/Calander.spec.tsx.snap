// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Calendar /> should match snapshot 1`] = `
<div
  className="calendarContainer "
  data-test="calendarContainer"
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    className="calendarTitle "
    data-test="calendarTitle"
  />
  <div
    className="calendarDaysContainer "
    data-test="calendarDaysContainer"
  >
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Sun
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Mon
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Tue
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Wed
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Thu
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Fri
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Sat
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    />
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    />
    <div
      className="calendarDayContainer calendarActiveDate "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum calendarActiveDateNumber"
        data-test="calendarNum"
      >
        1
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        2
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        3
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        4
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        5
         
      </p>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        6
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        7
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        8
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        9
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        10
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        11
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        12
         
      </p>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        13
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        14
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        15
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        16
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        17
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        18
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        19
         
      </p>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        20
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        21
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        22
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        23
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        24
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        25
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        26
         
      </p>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        27
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        28
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        29
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        30
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        31
         
      </p>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    />
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    />
  </div>
  
  
  
  
  
</div>
`;

exports[`<Calendar /> should match snapshot with data and titles 1`] = `
<div
  className="calendarContainer "
  data-test="calendarContainer"
  style={
    Object {
      "width": "100%",
    }
  }
>
  <div
    className="calendarTitle "
    data-test="calendarTitle"
  >
    ya alla
  </div>
  <div
    className="monthTitle "
    data-test="monthTitle"
  >
    October 2019
  </div>
  <div
    className="calendarDaysContainer "
    data-test="calendarDaysContainer"
  >
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Sun
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Mon
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Tue
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Wed
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Thu
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Fri
    </div>
    <div
      className="calendarHeaderContainer "
      data-test="calendarHeaderContainer"
    >
      Sat
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer calendarActiveDate "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum calendarActiveDateNumber"
        data-test="calendarNum"
      >
        1
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        <li
          className="dayDataItem "
          data-test="dayDataListItem"
        >
          item 1
        </li>
        <li
          className="dayDataItem "
          data-test="dayDataListItem"
        >
          item 2
        </li>
        <li
          className="dayDataItem "
          data-test="dayDataListItem"
        >
          item 3
        </li>
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        2
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        <li
          className="dayDataItem "
          data-test="dayDataListItem"
        >
          item 4
        </li>
        <li
          className="dayDataItem "
          data-test="dayDataListItem"
        >
          item 5
        </li>
        <li
          className="dayDataItem "
          data-test="dayDataListItem"
        >
          item 6
        </li>
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        3
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        4
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        5
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        6
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        7
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        8
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        9
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        10
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        11
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        12
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        13
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        14
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        15
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        16
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        17
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        18
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        19
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        20
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        21
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        22
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        23
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        24
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        25
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        26
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
  </div>
  <div
    className="calendarRow "
    data-test="calendarRowContainer"
  >
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        27
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        28
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        29
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        30
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <p
        className="calendarNum "
        data-test="calendarNum"
      >
        31
         
      </p>
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
    <div
      className="calendarDayContainer  "
      data-test="calendarDayContainer"
      onClick={[Function]}
      style={
        Object {
          "backgroundColor": "",
        }
      }
    >
      <ul
        className="dayDataList "
        data-test="dayDataList"
      >
        
      </ul>
    </div>
  </div>
  
  
  
  
  
</div>
`;
